src/features/Classes/ClassesTable/columns.jsx

```diff
@@ -118,6 +119,8 @@ const columns = [
       const dispatch = useDispatch();
       const institution = useSelector((state) => state.main.selectedInstitution);
       const deletionState = useSelector((state) => state.courses.newClass.status);
+      const [showToast, setShowToast] = useState(false);
+      const [toastMessage, setToastMessage] = useState('');
```

Comment:
Or even better, could be a custom hook to handle the toast status, what do u think about it?

Reply:
I added in a hook